superset/assets/visualizations/big_number.js

```diff
@@ -3,6 +3,15 @@ import { formatDate } from '../javascripts/modules/dates';
 
 require('./big_number.css');
 
+function getNumTicks(data) {
```

Comment:
The number of ticks should probably be a function of the width and/or height. The slice object has a `.width()` method you can use